network_analysis/offline-routing/README.md

```diff
@@ -1,36 +1,38 @@
-# Offline Routing
+# Offline routing
 
-Solve a route on-the-fly using only offline data.
+Solve a route on-the-fly using offline data.
 
-![](OfflineRouting.gif)
+![Image of offline routing](OfflineRouting.gif)
+
+## Use case
+
+You can use an offline network to enable routing in disconnected scenarios. For example, you could provide offline location capabilities to field workers repairing critical infrastructure in a disaster when network availability is limited.
 
 ## How to use the sample
 
-Left-click near a road to add a stop to the route. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Use the combo box in the top left corner to choose between the travel modes "Fastest" and "Shortest" (how the route is optimized). To move a stop, right-click the graphic to select it, then move your mouse to reposition, and finally right-click again to set the new position. The route will update on-the-fly while moving stops. The green box marks the boundary of the route geodatabase.
+Click near a road to start adding a stop to the route, click again to place it on the map. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Choose "Fastest" or "Shortest" from the drop down menu to control how the route is optimized. Use a secondary click on stops to select them, and then primary-click to re-place them. The route will update on-the-fly while moving stops. The green box marks the boundary of the routable area provided by the offline data.
```

Comment:
this section isn't quite right, partially reverting back to original instructions
```suggestion
Left-click near a road to add a stop to the route. A number graphic will show its order in the route. After adding at least 2 stops, a route will display. Choose "Fastest" or "Shortest" from the drop down menu to control how the route is optimized. To move a stop, right-click the graphic to select it, move your mouse to reposition and finally right-click again to set the new position. The route will update on-the-fly while moving stops. The green box marks the boundary of the routable area provided by the offline data.
```